src/flamegraph/svg.rs

```diff
@@ -114,9 +111,18 @@ where
     svg.write_event(Event::Start(
         BytesStart::borrowed_name(b"style").with_attributes(iter::once(("type", "text/css"))),
     ))?;
-    svg.write_event(Event::Text(BytesText::from_plain_str(
-        ".func_g:hover { stroke:black; stroke-width:0.5; cursor:pointer; }",
-    )))?;
+
+    let titlesize = &opt.font_size + 5;
+    svg.write_event(Event::Text(BytesText::from_escaped_str(&format!(
+        "
+text {{ font-family:{}; font-size:{}px; fill:rgb(0,0,0); }}
+#title {{ text-anchor:middle; font-size:{}px; }}
+{}",
+        &opt.font_type,
```

Comment:
This probably needs to be quoted and escaped.

Reply:
nice catch